docs/monthday.md

```diff
@@ -283,3 +283,17 @@ md = Temporal.MonthDay.from('08-24');
 Object.assign({}, md).day  // => undefined
 Object.assign({}, md.getFields()).day  // => 24
 ```
+
+### monthDay.**getISOCalendarFields()**(): { month: number, day: number }
```

Comment:
Here and below: It needs to include the refYear/refDay.  Include it either in this PR or in your refYear PR.

Reply:
I'll rebase the refYear/refDay PR on top of this one.